example/res/values/styles.xml

```diff
@@ -34,7 +34,6 @@
     <style name="Save" parent="@style/Button">
         <item name="android:layout_width">160dp</item>
         <item name="android:layout_marginTop">4dp</item>
-        <item name="android:text">@string/save</item>
```

Comment:
Putting strings inside a style definition is a weird thing to do, and also makes the style way less reusable.